docs/content/latest/explore/ysql-language-features/tablespaces.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Tablespaces
+linkTitle: Tablespaces
+description: Tablespaces in YSQL
+headcontent: Tablespaces in YSQL
+image: /images/section_icons/secure/create-roles.png
+menu:
+  latest:
+    identifier: explore-ysql-language-features-tablespaces
+    parent: explore-ysql-language-features
+    weight: 300
+isTocNested: true
+showAsideToc: true
+---
+
+This document describes how use tablespaces in YSQL.
+
+## Overview
+
+YSQL tablespaces are entities that specify how data associated with them should be replicated and distributed across cloud, regions, and zones.
+
+In PostgreSQL, tablespaces are used for specifying a location on disk with options to control data access. That is, tablespaces group tables and indexes based on how you intend to store and access the data. For instance, you can choose to place heavily accessed smaller tables and indexes in SSD or “fast” storage compared to other tables.
+
+For YugabyteDB clusters, however, location does not pertain to disk locations. For a cloud-native distributed database, location pertains to the cloud, regions, and zone where the data is supposed to be. Therefore, although YSQL tablespaces are built on PostgreSQL tablespaces that allow you to specify placement of data at a table level, this placement information in YSQL defines the number of replicas for a table and index, as well as how they can be distributed across a set of cloud, regions, and zones. 
+
+## Defining a Tablespace
+
+You can define a tablespace using the following syntax:
+
+```sql
+CREATE TABLESPACE tablespace_name 
+  [ OWNER { username | CURRENT_USER | SESSION_USER } ]
+  WITH ( replica_placement = placement_policy_json );
```

Comment:
Is there a way to highlight "placement_policy_json", "username" and "tablespace_name"? I am not sure of the format followed in docs... but wanted to know if we have a separate representation for Syntax vs User input parameters

Reply:
We just use italics.